.github/workflows/netlify.yml

```diff
@@ -0,0 +1,59 @@
+name: Visual regression tests with Netlify
+
+on:
+  pull_request_target:
+    types: [labeled]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  tests:
+
+    runs-on: ubuntu-latest
+
+    if: |
+      ( github.repository == 'optuna/optuna-visualization-regression-tests' ) &&
+      (
+         contains(github.event.pull_request.labels.*.name, 'safe-to-run-visual-regression-tests') ||
+         ( github.event_name == 'workflow_dispatch' )
+      )
+
+    steps:
+
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+
+    - name: Install Optuna
+      run: |
+        pip install -U pip
+        pip install --progress-bar off .
+        pip install git+https://github.com/optuna/optuna.git
+    - name: Install Optuna visual regression
```

Comment:
nit: I think you can use actions/cache like below to cache [the dataset of FasionMNIST](https://github.com/optuna/optuna-visualization-regression-tests/blob/5182483cb84c6c3dd401573cf98d8f197c83a174/studies.py#L244-L255).

https://github.com/optuna/optuna-visualization-regression-tests/blob/5182483cb84c6c3dd401573cf98d8f197c83a174/.github/workflows/gh-pages.yml#L24-L29

Reply:
Ah, this step actually cache the FashionMNIST dataset, not kurobako CLI. The step name may be a bit confusing...